#164583983 Travis Build Fail

ola357 opened the pull request:
#### What does this PR do?
Implement LFA feedback on:
-naming convention
-adding more test to test suite
-Travis CI build error

#### Description of Task to be completed?
-renamed files and classes
-added more tests

#### How should this be manually tested?
npm test

#### What are the relevant pivotal tracker stories?
#164583983
https://www.pivotaltracker.com/n/projects/2315133

ola357 committed:
feat(developer): I installed eslint
-extends the airbnb style guide
[Delivers #164486707]

ola357 committed:
feat(developer): setup babel
-to transpile from es6 to es5
[Delivers #164486291]

ola357 committed:
feat(developer): set up mocha
-for unit testing
[Delivers #164486992]

ola357 committed:
feat(user): setup the server using express framework
[Delivers #164487943]

ola357 committed:
feat(developer): the get all recieved messages
-currently failing its unit test
-following the Test Driven Development approach
[Delivers #164516793]

ola357 committed:
feat(user): get message route
-fetches all recieved messages for the user
[Delivers #164487450]

ola357 committed:
feat(developer): integrated travis
-for continous integration and
-integrated tests
[Delivers #164510155]

ola357 committed:
feat(developer): itegrated coveralls
-for test code coverage reporting
[Delivers #164516851]

ola357 committed:
feat(user): The user signup endpoint is built
-7 other endpoints are built
-tested with npm test and tests passing
-tested the api endpoints with postman
[Delivers #164487368]

ola357 committed:
feat(developer): restructured folders to follow MVC
-added CI badges from Travis and Coveralls to README
[Delivers #164537581]

ola357 committed:
feat(developer): Implemented LFA feedback
-Used ES6 classes and static methods
-Used MVC structure
-Destructured req.body in my POST routes
-prepared the app for hosting on heroku
[Delivers #164552962]

ola357 committed:
fixing error

ola357 committed:
fixing error 2
-changed the start script
-after npm run build & npm run serve

ola357 committed:
fixing error 3
-required @babel/polyfill
-in authControllers.js

ola357 committed:
fixing error 4
-added require @babel/register

ola357 committed:
fixing error 5
removed the required @babel polyfill & register
-added require @babelregister to app.js

ola357 committed:
fixing error 6
-added @babelcore

ola357 committed:
feat(developer): changed imported babelpolyfill
-in app.js

ola357 committed:
feat(developer): Implemented LFA feedback
_made classes Start with Capital Letter
-added more tests

ola357 committed:
fix(developer): Trying to fix error I get on Travis
-test are succesfully locally
-but they fail in travis
-just added clean to my scripts

ola357 committed:
pls work

ola357 committed:
plllsss  work

ola357 committed:
fix(build): change file naming convention
-implement LFA feedback
-add more test
-Capital letter for naming convention for classes
[Fixes #164583983]

cwizard2011 commented on dist/app.js:
```diff
@@ -0,0 +1,42 @@
+"use strict";
```
Add this folder to gitignore

cwizard2011 commented on dist/controllers/authControllers.js:
```diff
@@ -0,0 +1,205 @@
+"use strict";
```
add the dist folder to gitignore

cwizard2011 commented on server/controllers/messagesController.js:
```diff
@@ -0,0 +1,88 @@
+import messages from '../models/messages';
```
Looks nice, add comments to your code

ola357 committed:
feat(rename) implement LFA feedback
-change naming convention of file
-add comments to code

ola357 committed:
feat(developer): complete challenge 2
-host on heroku
-implement lfa feedback
-ready to merge to develop
[Delivers #164620539]

ola357 committed:
Merge pull request #7 from ola357/ft-swagger-documentation-164620539

#164620539 Swagger Documentation

ola357 committed:
feat(developer) edit the README.md file

ola357 committed:
Merge branch 'bg-travis-build-failing-164583983' of https://github.com/ola357/epic-mail into boom3

ola357 committed:
feat(travis) fix the travis build fail
-error: cannot find module
-change the filenames to correspond to import names
